composer.json

```diff
@@ -6,35 +6,33 @@
     "minimum-stability": "dev",
     "prefer-stable": true,
     "require": {
-        "php": ">=7.2",
-        "drupal/core": "^8.7",
-        "drupal/language_selection_page": "2.3",
+        "php": ">=7.3",
+        "drupal/core": "^8.9 || ^9",
+        "drupal/language_selection_page": "^2.7",
         "drupal/pathauto": "^1.6"
     },
     "require-dev": {
         "composer/installers": "~1.5",
         "consolidation/annotated-command": "^2.8.2",
-        "consolidation/robo": "~1.4",
-        "drupal/core-composer-scaffold": "^8.8",
+        "consolidation/robo": "^1.4.13 || ^2",
+        "drupal/core-composer-scaffold": "^8.9 || ^9",
         "drupal/config_devel": "~1.2",
-        "drupal/devel": "~1.2",
+        "drupal/devel": "~4.1",
         "drupal/drupal-extension": "~4.0",
-        "drupal/redirect": "^1.4",
-        "drush/drush": "~9.0@stable",
-        "guzzlehttp/guzzle": "~6.3",
+        "drupal/redirect": "^1.6",
+        "drush/drush": "~10.0",
         "openeuropa/behat-transformation-context": "~0.1",
-        "openeuropa/code-review": "^1.3.0",
-        "openeuropa/drupal-core-require-dev": "^8.7",
-        "openeuropa/task-runner": "~1.0.0-beta6",
-        "phpunit/phpunit": "~6.0",
-        "symfony/browser-kit": "~3.0 || ~4.0"
+        "openeuropa/code-review": "~1.6",
+        "openeuropa/drupal-core-require-dev": "^8.9 || ^9",
+        "openeuropa/task-runner-drupal-project-symlink": "^1.0",
+        "phpunit/phpunit": "^7 || ^8"
     },
     "_readme": [
         "We explicitly require consolidation/robo to allow lower 'composer update --prefer-lowest' to complete successfully.",
         "We explicitly require consolidation/annotated-command to allow lower 'composer update --prefer-lowest' to complete successfully."
     ],
     "scripts": {
-        "post-install-cmd": "./vendor/bin/run drupal:site-setup",
+        "post-install-cmd": "./vendor/bin/run drupal:site-setup drupal:symlink-project",
```

Comment:
`drupal:site-setup` calls already `drupal:symlink-project` so no neeed to put it here.